artwork/README.md

```diff
@@ -2,26 +2,26 @@
 
 ---
 
-# Logisim-evolution logo project #
+# Logisim-evolution project logo #
 
 Designed by Marcin Orlowski <http://MarcinOrlowski.com>
 
 ## SVG files ##
 
 * `logisim-evolution-logo-src.svg` - main logo project file
-* 'logisim-evolution-logo.svg` - derived from main project file with all texts converted to paths.
+* `logisim-evolution-logo.svg` - derived from main project file with all texts converted to paths.
 
-NOTE: to properly render `logisim-evolution-logo-src.svg` (be it in-app, on webpage etc) you must
-have all the used fonts installed on target machine or your renderer/browser substitute fonts
-producing wrong image. Alternatively you can either render your image a bitmap (i.e. PNG) or
-convert all texts to paths (Inkscape: "Object" menu -> "Object to Path"). We provide such version
-of logo SVG as `logisim-evolution-logo.svg`.
+NOTE: to properly render the `logisim-evolution-logo-src.svg` (be it in-app, on the web page or elsewhere)
+one must have all the used fonts installed on target machine otherwise your renderer/browser will substitute
+the fonts using available ones which resulting in incorrect final image. Alternatively you can either render
+your image to a bitmap (i.e. PNG) or  convert all texts to paths (in Inkscape: "Object" menu -> "Object to Path").
+This will however make SVG file bigger ( see `logisim-evolution-logo.svg` which is exactly such font-less version).
 
```

Comment:
How about the following wording?

"NOTE: To properly render the `logisim-evolution-logo-src.svg` (be it in-app, on the web page, or elsewhere)
 one must have all the used fonts installed on the build machine. Otherwise, your renderer/browser will substitute
 the fonts using available ones, which result in a poor final image. Alternatively, you can either render
 your image to a pixmap (preferably in PNG format) or  convert all texts to paths (in Inkscape: "Object" menu -> "Object to Path"). However, this will make the SVG file bigger (see `logisim-evolution-logo.svg`, which is exactly such font-less version)."

Reply:
For me, "poor final image" indicates rather quality issues (like high compression rate based etc) not incorrect imagery. I'd go with "(PNG format is recommended, due to its lossless compression and transparency support)" which is in fact the reasoning why PNG is mentioned. "pixmap" => "bitmap". Never seen a "pixmap" used in such context. So that would go like this:

---

"NOTE: To properly render the `logisim-evolution-logo-src.svg` (be it in-app, on the web page, or elsewhere)
one must have all the used fonts installed on the build machine. Otherwise, your renderer/browser will substitute
the fonts using available ones, which result in a incorrect final image. Alternatively, you can either render
your image to a bitmap (PNG image format is recommended, due to its lossless compression and transparency support) or convert all texts to paths (in Inkscape: "Object" menu -> "Object to Path"). However, this will make the SVG file bigger (see logisim-evolution-logo.svg, which is exactly such font-less version)."

---

I updated the PR to be like the above. We can always roll back if @davidhutchens object.